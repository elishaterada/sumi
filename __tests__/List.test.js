import React from 'react';
import renderer from 'react-test-renderer';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';

import {
  DLClass,
  DDClass,
  DTClass,
} from '../src/components/List';

expect.addSnapshotSerializer(createSerializer(emotion));

test('renders List', () => {
  const reactComponent = renderer
    .create(
      <dl className={DLClass}>
        <dt className={DTClass}>Espresso</dt>
        <dd className={DDClass}>1 oz. espresso</dd>
        <dt className={DTClass}>Espresso Macchiato</dt>
        <dd className={DDClass}>1 oz. espresso + 1 Tbs. foamed milk</dd>
        <dt className={DTClass}>Cappuccino</dt>
        <dd className={DDClass}>1/3 espresso, 1/3 steamed milk, 1/3 foam</dd>
      </dl>,
    )
    .toJSON();

  expect(reactComponent).toMatchSnapshot();
});
