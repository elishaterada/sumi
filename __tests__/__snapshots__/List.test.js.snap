// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders List 1`] = `
.emotion-6 {
  margin: 0 0 1rem;
}

.emotion-0 {
  font-weight: 600;
  margin: 0 0 0.5rem;
  color: #333333;
}

@media (min-width:64em) {
  .emotion-0 {
    float: left;
    width: 12rem;
    clear: left;
    overflow: hidden;
  }
}

.emotion-1 {
  margin: 0 0 1rem;
  color: #666666;
}

@media (min-width:64em) {
  .emotion-1 {
    margin-left: 15rem;
  }
}

<dl
  className="emotion-6"
>
  <dt
    className="emotion-0"
  >
    Espresso
  </dt>
  <dd
    className="emotion-1"
  >
    1 oz. espresso
  </dd>
  <dt
    className="emotion-0"
  >
    Espresso Macchiato
  </dt>
  <dd
    className="emotion-1"
  >
    1 oz. espresso + 1 Tbs. foamed milk
  </dd>
  <dt
    className="emotion-0"
  >
    Cappuccino
  </dt>
  <dd
    className="emotion-1"
  >
    1/3 espresso, 1/3 steamed milk, 1/3 foam
  </dd>
</dl>
`;
